README.md

```diff
@@ -4,15 +4,16 @@
 
 ### Prerequisites
 1. Install Ansible on your control machine, please refer to [Installing Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
-2. Install required Python libraries in requirements.txt
+2. Install Tesseract Open Source OCR Engine for extracting text from screenshots, please refer to [Installing Tesseract](https://github.com/tesseract-ocr/tesseract?tab=readme-ov-file#installing-tesseract)
```

Comment:
Can we test this tool firstly? If not installed then this feature is disabled, else can use this feature for info extracting from screenshot.

Reply:
Sure. Will add it.